sklearn/utils/estimator_checks.py

```diff
@@ -4784,8 +4786,11 @@ def check_param_validation(name, estimator_orig):
 
             with raises(InvalidParameterError, match=match, err_msg=err_msg):
                 if tags.target_tags.one_d_labels or tags.target_tags.two_d_labels:
-                    # The estimator is a label transformer and take only `y`
-                    getattr(estimator, method)(y)
+                    if tags.target_tags.multi_output:
+                        getattr(estimator, method)(X, y)
+                    else:
+                        # The estimator is a label transformer and take only `y`
+                        getattr(estimator, method)(y)
```

Comment:
probably @jeremiedbb knows better as why this line is not covered here, but the one bellow is.

Reply:
For me this is a false positive because locally `LabelBinarizer` in this PR will get through that branch and raise:

```pytb
E               sklearn.utils._param_validation.InvalidParameterError: The 'neg_label' parameter of LabelBinarizer must be an instance of 'int'. Got <sklearn.utils.estimator_checks.BadType object at 0x13ea44500> instead.
```